user/timepix3/module/src/Timepix3Event2StdEventConverter.cc

```diff
@@ -56,11 +66,23 @@ bool Timepix3TrigEvent2StdEventConverter::Converting(eudaq::EventSPC ev, eudaq::
   timestamp = timestamp_raw + (static_cast<long long int>(m_TDCoverflowCounter) << 35);
 
   // Calculate timestamp in picoseconds assuming 320 MHz clock:
-  uint64_t triggerTime = (timestamp + static_cast<long long int>(stamp) / 12) * 3125;
+  uint64_t triggerTime = timestamp * 3125 +(stamp * 3125) / 12;
 
   // Set timestamps for StdEvent in nanoseconds (timestamps are picoseconds):
   d2->SetTimeBegin(triggerTime);
   d2->SetTimeEnd(triggerTime);
+
+  // Identify the detetor type
+  d2->SetDetectorType("SpidrTrigger");
+
+  // Create a StandardPlane representing one DUMMY sensor plane
```

Comment:
Do we need this really? Feels odd.

Reply:
I had no other idea how to ship this info into corry.